FROM: Ian Lea <ian....@gmail.com>
SUBJECT: Re: An unexpected network error occurred
DATE: 10 May 2011

A full stack trace dump is always helpful.  Are the three instances on
one server with a local index  directory, or on different servers
accessing a network drive (how?) or what?  If the index is locked it
would be surprising that you could update it from 2 of the instances.


--
Ian.


On Tue, May 10, 2011 at 1:05 PM, Yogesh Dabhi <ydabhi@asite.com> wrote:
> Three Instance of My application & lucene index directory shared for all
> instance
>
> Lucene version 3.1
>
> Lock factory:-  NativeFSLockFactory
>
> Instance1 jdk64 ,64 os
>
> Instance2 jdk64 ,64 os
>
> Instance3 jdk32 ,32 os
>
>
>
> When I try to search the data from  the index directory  from Instance1
> I got bellow error
>
> An unexpected network error occurred
>
> In lucene directory there write.lock file
>
>
>
> I cannot read data & update data in index from Instance1
>
> But for other two Instances its working fine
>
> is there a way to handle such error
>
> Thanks & Regards
>
> Yogesh
>
>

---------------------------------------------------------------------
To unsubscribe, e-mail: java-user-unsubscribe@lucene.apache.org
For additional commands, e-mail: java-user-help@lucene.apache.org


